### ISSUE TITLE
Interdependency 

### ISSUE THREAD

**Issue by weaversa** [GaloisInc/cryptol-specs#3]
I can imagine someone coming here to get a spec for algorithm X, but needing to carry the whole library along because of Common.

Wouldn't it be better if each spec was standalone (all reuse came from the Cryptol prelude)?

Also, I would encourage each spec to be paired with the original paper specification (pdf) and tests  taken from the same.

**Comment by brianhuffman** [comment]
I agree. I think a common use-case for this repo would be that someone just wants to grab an implementation for one specific algorithm to use in a local project. We should make it easy to copy out a file or two for standalone use.
